chord_metadata_service/chord/tests/test_api.py

```diff
@@ -95,6 +95,7 @@ def test_create_dataset(self):
             self.assertEqual(Dataset.objects.count(), len(self.valid_payloads))
 
     def test_dats(self):
+        self.dats_valid_payload['dats_file'] = json.dumps({})
```

Comment:
instead of mutating the test property, create a new payload dictionary with `dats_file` property changed:

```py
payload = {**self.dats_valid_payload, 'dats_file': json.dumps({})}
```